tests/unit/server.test.js

```diff
@@ -81,6 +81,74 @@ test.cb('that calling close will trigger the onclose of websockets', t => {
   };
 });
 
+test.cb('that calling close on the server without a `code` will result in a default code of `1000`', t => {
```

Comment:
I think it's better to move this test to `functional/close-algorithm.test.js`

Reply:
Every test in `functional/close-algorithm.test.js` calls close on the _client_. That's why I put the test in here – alongside other tests that call close on the _server_.